app/containers/message/Components/CollapsibleQuote.tsx

```diff
@@ -0,0 +1,183 @@
+import { transparentize } from 'color2k';
+import { dequal } from 'dequal';
+import React, { useContext, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { themes } from '../../../constants/colors';
+import { IAttachment } from '../../../definitions/IAttachment';
+import { TGetCustomEmoji } from '../../../definitions/IEmoji';
+import { CustomIcon } from '../../../lib/Icons';
+import sharedStyles from '../../../views/Styles';
+import Markdown from '../../markdown';
+import MessageContext from '../Context';
+import Touchable from '../Touchable';
+
+const styles = StyleSheet.create({
+	button: {
+		flex: 1,
+		flexDirection: 'row',
+		alignItems: 'center',
+		marginTop: 6,
+		alignSelf: 'flex-start',
+		borderWidth: 1,
+		borderRadius: 4
+	},
+	attachmentContainer: {
+		flex: 1,
+		borderRadius: 4,
+		flexDirection: 'column',
+		padding: 8
+	},
+	authorContainer: {
+		flex: 1,
+		flexDirection: 'row',
+		alignItems: 'center'
+	},
+	fieldsContainer: {
+		flex: 1,
+		flexWrap: 'wrap',
+		flexDirection: 'row'
+	},
+	fieldContainer: {
+		flexDirection: 'column',
+		padding: 10
+	},
+	fieldTitle: {
+		fontSize: 15,
+		...sharedStyles.textBold
+	},
+	marginTop: {
+		marginTop: 4
+	},
+	marginBottom: {
+		marginBottom: 4
+	},
+	title: {
+		flex: 1,
+		fontSize: 16,
+		...sharedStyles.textMedium
+	},
+	touchableContainer: { flex: 1, flexDirection: 'row' },
+	markdownFontSize: { fontSize: 15 },
+	iconContainer: { width: 20, height: 20, right: 8, top: 6, justifyContent: 'center', alignItems: 'center' }
+});
+
+interface IMessageFields {
+	attachment: IAttachment;
+	theme: string;
+	getCustomEmoji: TGetCustomEmoji;
+}
+
+interface IMessageReply {
+	attachment: IAttachment;
+	timeFormat?: string;
+	index: number;
+	theme: string;
+	getCustomEmoji: TGetCustomEmoji;
+}
+
+const Fields = React.memo(
+	({ attachment, theme, getCustomEmoji }: IMessageFields) => {
+		if (!attachment.fields) {
+			return null;
+		}
+
+		const { baseUrl, user } = useContext(MessageContext);
+		return (
+			<View style={styles.fieldsContainer}>
+				{attachment.fields.map(field => (
+					<View key={field.title} style={[styles.fieldContainer, { width: field.short ? '50%' : '100%' }]}>
+						<Text testID='collapsibleQuoteTouchableFieldTitle' style={[styles.fieldTitle, { color: themes[theme].bodyText }]}>
+							{field.title}
+						</Text>
+						<Markdown
+							msg={field?.value || ''}
+							baseUrl={baseUrl}
+							username={user.username}
+							getCustomEmoji={getCustomEmoji}
+							theme={theme}
+							style={[styles.markdownFontSize]}
+						/>
+					</View>
+				))}
+			</View>
+		);
+	},
+	(prevProps, nextProps) =>
+		dequal(prevProps.attachment.fields, nextProps.attachment.fields) && prevProps.theme === nextProps.theme
+);
+
+const CollapsibleQuote = React.memo(
+	({ attachment, index, getCustomEmoji, theme }: IMessageReply) => {
```

Comment:
We should receive `collapsed` as prop, so it can initialize open or closed
https://github.com/RocketChat/Rocket.Chat/blob/7632f12cfcc7ed8ee8f843587fdff63b101cc765/client/components/Message/Attachments/hooks/useCollapse.tsx#L10
https://github.com/RocketChat/Rocket.Chat/blob/7632f12cfcc7ed8ee8f843587fdff63b101cc765/client/components/Message/Attachments/Attachment/Collapse.tsx#L8